src/sketch.js

```diff
@@ -5,54 +5,108 @@
 // sound controller container
 let soundOn = false;
 
-// perlin noise terrain container
-let terrain;
-
 // soundtrack p5.SoundFile container
 let soundtrack;
 
+// webcam stream container
+let webcam;
+
+// webcam controller container
+let webcamOn = false;
+
+// perlin noise terrain container
+let terrain;
+
 /**
  * I draw this sketch sound indicator.
```

Comment:
Please update the documentation. This is no longer to draw only sound indicator. 👍